src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -152,9 +166,16 @@ const flattenBreadcrumbsItems = (children: React.ReactNode) => {
     },
   );
   if (process.env.NODE_ENV !== 'production' && shouldThrowError) {
-    throw 'Only <Breadcrumbs.Item>, <BreadcrumbsItem>, or React.Fragment of aforementioned components allowed';
+    throw 'Only <Breadcrumbs.Item> or React.Fragment of aforementioned components allowed';
```

Comment:
We should use an `Error` object when throwing, for consistency. I think all the other `throw`s do. Or should...

Reply:
Changed to throwing `new Error`